count_chair.py

```diff
@@ -0,0 +1,10 @@
+str1 = "ala ma kota"
+char_dict = dict()
+
+for char in str1:
+    count = str1.count(char)
+    char_dict[char] = count
+
+print('Result:',char_dict)
+
```

Comment:
Powinniśmy dostać tylko wynik z największą liczbą wystąpień i liczba wystąpień, a dwa to napiszmy to w funkcji

Reply:
ok wiem, chciałem przecwiczyc jak ta scieżka idzie

Reply:
naprawie to w zabawie z nastepnymi commitami,